packages/playground-ui/theme.css

```
@@ -0,0 +1,379 @@
+  --height-table-row: 56px;
+  --height-table-row-small: 40px;
+  --height-badge-default: 28px;
+  --height-avatar-sm: 24px;
+  --height-avatar-md: 32px;
+  --height-avatar-lg: 40px;
+  --height-dropdown-max-height: 300px;
+
+  --max-height-icon-default: 16px;
+  --max-height-icon-lg: 20px;
+  --max-height-icon-sm: 12px;
+  --spacing-form-sm: 1.75rem;
```

Stylelint (17.12.0)

[error] Duplicate custom property "--spacing-form-sm" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)